src/bundle/Resources/views/content/tabs/content.html.twig

```diff
@@ -0,0 +1,22 @@
+{% set languageCode = content.versionInfo.contentInfo.mainLanguageCode %}
+
+{% for contentTypeGroup in contentType.contentTypeGroups %}
```

Comment:
Not the right element :-)

This is the group(s) the Content Type is part of. You want to iterate on the FieldDefinitions groups, that can be accessed with `$fieldDefinition->fieldGroup`.

You can use `@ezpublish.fields_groups.list` to get the human readable strings (uses translations).